src/OpenTelemetry.Exporter.OpenTelemetryProtocol/Implementation/LogRecordExtensions.cs

```diff
@@ -79,7 +83,7 @@ internal static OtlpLogs.LogRecord ToOtlpLog(this LogRecord logRecord, SdkLimitO
 
             var attributeValueLengthLimit = sdkLimitOptions.AttributeValueLengthLimit;
             var attributeCountLimit = sdkLimitOptions.AttributeCountLimit ?? int.MaxValue;
-
+#if EXPOSE_EXPERIMENTAL_FEATURES
```

Comment:
Are we still going to rename the EventId keys? I think we should.

Reply:
PS: Probably worth mentioning this in CHANGELOG.

Reply:
> Are we still going to rename the EventId keys? I think we should.

See conversation here https://github.com/open-telemetry/opentelemetry-dotnet/pull/4762#discussion_r1290833563

> PS: Probably worth mentioning this in CHANGELOG.

Yes, please hold on merging this. I haven't submitted a changelog entry yet because I'm working on a to document our experimental features somewhat centrally.